webapp/platform/client/src/client4.ts

```diff
@@ -3490,8 +3490,9 @@ export default class Client4 {
         );
     };
 
-    getGroups = (filterAllowReference = false, page = 0, perPage = 10, includeMemberCount = false, hasFilterMember = false) => {
+    getGroups = (q = '', filterAllowReference = false, page = 0, perPage = 10, includeMemberCount = false, hasFilterMember = false) => {
```

Comment:
Why do we need to add this `q` argument? I failed to see where it is used.

Reply:
It's the filter that we set to query groups by name, it's used [here](https://github.com/mattermost/mattermost/pull/24292/files#diff-b0aca3f09658bed4447dad8edfe7dbeb52bed383d70cc15367689883251fafb8R1130)